Client/src/main/java/io/github/jwdeveloper/tiktok/http/HttpClientFactory.java

```diff
@@ -37,7 +37,9 @@ public HttpClientBuilder client(String url) {
         return new HttpClientBuilder(url, liveClientSettings.getHttpSettings().clone());
     }
 
+
     //Does not contains default httpClientSettings, Params, headers, etd
+    //Edit: Do we even use it?
```

Comment:
We do not and I doubt anyone wants to ignore the default headers we found to be required. It can honestly probably be removed.